src/extension/options.js

```diff
@@ -36,6 +36,11 @@ function isValidGitHubURL(giturl) {
     && Object.keys(getGitHubSettings(giturl)).length === 3;
 }
 
+function isValidProdHost(prodHost) {
+  const prodHostPattern = /^$|^(https?:\/\/)?([a-zA-Z0-9-]+\.)+[a-zA-Z]{2,}\/?/;
```

Comment:
I would not allow `https://` prefixes and trailing `/`s - we actually want domain/host names only